webserver/static/js/feverscreen.ts

```diff
@@ -1524,7 +1511,16 @@ window.onload = async function () {
     const temperature =
       GThermalRefTemp + (UncorrectedHotspot - UncorrectedThermalRef) * 0.01;
     //const temperature = 40
-    showTemperature(temperature, frameInfo);
+
+    if (GFaces.length) {
+      const face = GFaces[0];
```

Comment:
So if we only want to show this if the haar has detected a face can do 
`if ( face.haarActive() ) { `
or if we are to pick on try find one with an active haar first? otherwise just take the first face

Reply:
Done